test/test_sdk_installation.py

```diff
@@ -13,44 +13,78 @@
 import subprocess
 import sys
 
+# Constants
+
+SCRIPTS_PATH = os.path.dirname(__file__) + "/scripts/"
+
+# Helper functions
+
+def scripts_for(path):
+    return [f for f in os.listdir(SCRIPTS_PATH + path) if f.endswith(".py")]
+
+
+def scripts_path_on_vm(path):
+    return "/vagrant/test/scripts/" + path + "/"
```

Comment:
what path variable means here is ambiguous since it is also in the name. Maybe call it `scripts_dirname` or something instead?